src/Perl6/World.nqp

```diff
@@ -4882,6 +4882,21 @@ class Perl6::World is HLL::World {
         }
     }
 
+    method find_single_symbol_in_setting($name) {
+        my $no-outers := 0; # If 'true' then don't look in the outer contexts
+
+        my $setting_name := Perl6::ModuleLoader.transform_setting_name($!setting_name);
+        my $ctx := Perl6::ModuleLoader.load_setting($setting_name);
+
+        while $ctx {
```

Comment:
if `$no-outers` is gone, wouldn't this loop be completely unnecessary?

Reply:
Whoops, `$no-outers` was missed when I copied out the single-element logic from `find_symbol_in_setting`. The loop is still necessary, since it checks outer contexts until there aren't any left.